llvm_util/llvm2alive.cpp

```diff
@@ -74,6 +77,25 @@ class llvm2alive_ : public llvm::InstVisitor<llvm2alive_, unique_ptr<Instr>> {
     return a != 0 ? a : DL().getPrefTypeAlignment(ty);
   }
 
+  auto convert_constexpr(llvm::ConstantExpr *cexpr) {
+    llvm::Instruction *newI = cexpr->getAsInstruction();
+    static unsigned constexpr_idx = 0;
+    stringstream ss;
+    ss << "__constexpr_" << constexpr_idx++;
+    newI->setName(ss.str());
+
+    i_constexprs.push_back(newI);
+    auto ptr = this->visit(*newI);
+    auto i = ptr.get();
+    BB->addInstr(move(ptr));
```

Comment:
Adding to IR::BasicBlock (not llvm::BasicBlock) was needed because sym_exec failed with updating corresponding StateValue.